wren-ai-service/tools/providers-setup/ui_components.py

```
@@ -0,0 +1,516 @@
+                    st.error(llm_msg)
+
+
+
+def render_embedder_config():
+    """
+    Render the embedding model configuration section.
+    Displays current provider settings and allows users to override the model name,
+    alias, timeout, and base URL. Supports testing and saving.
+    """
+    with st.expander("Embedder Configuration", expanded=False):
+        st.markdown(f"**type:** `embedder`")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)